main.py

```diff
@@ -1,15 +1,26 @@
 import difflib
 import sys
 import time
+import argparse
+import os
 
 from utils.common import load_config, load_env
-from utils.constants import DIFFS_DIR, START_TIME
+from utils.constants import DIFFS_DIR, START_TIME, DEFAULT_CONFIG_PATH
 from utils.explorer import get_contract_from_explorer
 from utils.github import get_file_from_github, resolve_dep
 from utils.helpers import create_dirs
 from utils.logger import logger
 
 
+EXPLORER_API_TOKEN = load_env("ETHERSCAN_TOKEN", masked=True)
```

Comment:
might want to have multiple tokens (Optimism would require their own, Linea allows empty token for now, but they maintain a separate database and can change it in the future)

Reply:
Added to config field which specified token env variable name.
Btw, the token happens not to be required for Etherscan as well (at least for our number of requests)